### ISSUE TITLE
[question]: Unable to start receiver com.onesignal.notifications.receivers.BootUpReceiver:

### ISSUE THREAD

**Issue by bytemtek** [OneSignal/react-native-onesignal#1605]
### How can we help?

Hello,

I use latest version but I see this error in my Play Console.
How can I fix it ?
```
Exception java.lang.RuntimeException: Unable to start receiver com.onesignal.notifications.receivers.BootUpReceiver: java.lang.NullPointerException
  at android.app.ActivityThread.handleReceiver (ActivityThread.java:4903)
  at android.app.ActivityThread.-$$Nest$mhandleReceiver
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2420)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loopOnce (Looper.java:226)
  at android.os.Looper.loop (Looper.java:313)
  at android.app.ActivityThread.main (ActivityThread.java:8757)
  at java.lang.reflect.Method.invoke
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:571)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1067)
Caused by java.lang.NullPointerException:
  at com.onesignal.core.internal.application.impl.ApplicationService.getAppContext (ApplicationService.java)
  at com.onesignal.core.internal.device.impl.DeviceService.packageInstalledAndEnabled
  at com.onesignal.core.internal.device.impl.DeviceService.isGMSInstalledAndEnabled
  at com.onesignal.core.internal.device.impl.DeviceService.supportsGooglePush
  at com.onesignal.core.internal.device.impl.DeviceService.getDeviceType
  at com.onesignal.core.internal.device.impl.DeviceService.isAndroidDeviceType
  at com.onesignal.location.LocationModule$register$1.invoke
  at com.onesignal.location.LocationModule$register$1.invoke
  at com.onesignal.common.services.ServiceRegistrationLambda.resolve
  at com.onesignal.common.services.ServiceProvider.getServiceOrNull
  at com.onesignal.common.services.ServiceProvider.getService
  at com.onesignal.common.services.ServiceRegistrationReflection.resolve
  at com.onesignal.common.services.ServiceProvider.getServiceOrNull
  at com.onesignal.common.services.ServiceProvider.getService
  at com.onesignal.common.services.ServiceRegistrationReflection.resolve
  at com.onesignal.common.services.ServiceProvider.getServiceOrNull
  at com.onesignal.common.services.ServiceProvider.getService
  at com.onesignal.internal.OneSignalImp.initWithContext
  at com.onesignal.OneSignal.initWithContext
  at com.onesignal.notifications.receivers.BootUpReceiver.onReceive
  at android.app.ActivityThread.handleReceiver (ActivityThread.java:4894)
```
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jennantilla** [comment]
Hello @bytemtek apologies for the delay in getting back to you. Is this still an issue you are facing? If so, would you please provide the following:

- The SDK version you are using
- Reproduction steps
- [Full logs](https://documentation.onesignal.com/docs/capturing-a-debug-log#get-the-logs) capturing this behavior

Much appreciated! We look forward to assisting you!

**Comment by jkasten2** [comment]
@bytemtek thanks for reporting, this is a duplicate of issue #1587. The issue should be fixed in [Release 5.1.0](https://github.com/OneSignal/react-native-onesignal/releases/tag/5.1.0).

If you still see this issue after updating please follow up in issue #1587, thanks!

**Comment by uasghar-smile** [comment]
Hi @jkasten2  please check [this](https://github.com/OneSignal/react-native-onesignal/issues/1587#issuecomment-2009117498)
